composer.json

```diff
@@ -55,11 +56,11 @@
         "sonata-project/core-bundle": "^3.14",
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
-        "symfony-cmf/testing": "^2.1.12",
+        "symfony-cmf/testing": "dev-master as 2.1.12",
```

Comment:
should we tag version 3 of testing? it seems the missing piece is https://github.com/symfony-cmf/Testing/issues/166 but we could maybe ignore that for now.

Reply:
I'll better update to `"symfony-cmf/testing": "^3@dev"` and I'd leave it like that for now

Reply:
@dbu please do :pray: an extra release never hurts, relying on a dev version can.